integration/spark2/src/main/scala/org/apache/carbondata/indexserver/DataMapJobs.scala

```diff
@@ -42,7 +43,18 @@ class DistributedDataMapJob extends AbstractDataMapJob {
       LOGGER.debug(s"Size of message sent to Index Server: $messageSize")
     }
     val (resonse, time) = logTime {
-      IndexServer.getClient.getSplits(dataMapFormat).toList.asJava
+      val spark = SparkSQLUtil.getSparkSession
+      var taskGroupId = spark.sparkContext.getLocalProperty("spark.jobGroup.id")
+      var taskGroupDesc = spark.sparkContext.getLocalProperty("spark.job.description")
+      taskGroupId = taskGroupId match {
```

Comment:
change to 
 
val taskGroupDesc = spark.sparkContext.getLocalProperty("spark.job.description") match ...

same for taskGroupId

Reply:
done